### PR TITLE
test: Upgrade Cypress to 7.1.0

### PR DESCRIPTION
**Pull request by nalmabrcom**
Signed-off-by: Maryna Nalbandian <maryna.nalbandian@broadcom.com>

### CODE REVIEW

**Review by temanbrcom — APPROVED** [review]
Did you check the TPSR for it?

**Comment by nalmabrcom** [comment]
> Did you check the TPSR for it?

Hmm, seems it is not yet in https://clearlydefined.io/. I couldn't find it there.

File: tests/yarn.lock [hunk 1]
```diff
@@ -2597,46 +2607,46 @@ cypress-xpath@^1.6.1:
...
+  version "7.1.0"
+  resolved "https://registry.yarnpkg.com/cypress/-/cypress-7.1.0.tgz#6cb5dc22c6271a9d7a79a2477251a95afc77e531"
+  integrity sha512-AptQP9fVtN/FfOv8rJ9hTGJE2XQFc8saLHT38r/EeyWhzp0q/+P/DYRTDtjGZHeLTCNznAUrT4lal8jm+ouS7Q==
   dependencies:
     "@cypress/listr-verbose-renderer" "^0.4.1"
     "@cypress/request" "^2.88.5"
     "@cypress/xvfb" "^1.2.4"
-    "@types/sinonjs__fake-timers" "^6.0.1"
+    "@types/node" "^14.14.31"
+    "@types/sinonjs__fake-timers" "^6.0.2"
```
Comment (asatklichov) [inline tests/yarn.lock:2619]: Any idea if sinon should be checked on TPRS or rely on check for cypress. 
It is a separate lib. at the end.

**Comment by grianbrcom** [comment]
Probably, we can use the information from npmjs https://www.npmjs.com/package/cypress/v/7.1.0
There, cypress has an MIT license which is OK.

File: tests/yarn.lock (hunk 1 above)
Comment (grianbrcom) [reply tests/yarn.lock:2619]: I think no because `sinon` is a dependency for `cypress`.

**Review by grianbrcom — APPROVED** [review]
Looks good, but I have no idea what to do if we didn't found the dependency in the clearlydefined.

**Review by asatklichov — APPROVED** [review]
Ⓜ️

**Pull request merged**